### PR TITLE
Add all available Revolut currencies

### PR DESCRIPTION
**Pull request by ghost**
Fixies https://github.com/bisq-network/bisq/issues/4273

### CODE REVIEW

File: core/src/main/java/bisq/core/locale/CurrencyUtil.java [hunk 1]
```diff
@@ -267,32 +267,42 @@ public static void setBaseCurrencyCode(String baseCurrencyCode) {
...
-                new FiatCurrency("NZD"),
-                new FiatCurrency("TRY"),
-                new FiatCurrency("ILS"),
-                new FiatCurrency("AED"),
-                new FiatCurrency("CAD"),
+                new FiatCurrency("HRK"),
                 new FiatCurrency("HUF"),
+                new FiatCurrency("HUF"),
+                new FiatCurrency("ILS"),
                 new FiatCurrency("INR"),
```
Comment (sqrrm) [inline core/src/main/java/bisq/core/locale/CurrencyUtil.java]: I don't see `INR` in their supported currencies.

File: core/src/main/java/bisq/core/locale/CurrencyUtil.java [hunk 2]
```diff
@@ -267,32 +267,42 @@ public static void setBaseCurrencyCode(String baseCurrencyCode) {
...
                 new FiatCurrency("HKD"),
-                new FiatCurrency("AUD"),
-                new FiatCurrency("NZD"),
-                new FiatCurrency("TRY"),
-                new FiatCurrency("ILS"),
-                new FiatCurrency("AED"),
-                new FiatCurrency("CAD"),
+                new FiatCurrency("HRK"),
                 new FiatCurrency("HUF"),
+                new FiatCurrency("HUF"),
```
Comment (sqrrm) [inline core/src/main/java/bisq/core/locale/CurrencyUtil.java]: Duplicate

File: core/src/main/java/bisq/core/locale/CurrencyUtil.java [hunk 3]
```diff
@@ -267,32 +267,42 @@ public static void setBaseCurrencyCode(String baseCurrencyCode) {
...
+                new FiatCurrency("ILS"),
                 new FiatCurrency("INR"),
+                new FiatCurrency("ISK"),
+                new FiatCurrency("JPY"),
```
Comment (sqrrm) [inline core/src/main/java/bisq/core/locale/CurrencyUtil.java:287]: Duplicate

**Review by sqrrm — CHANGES_REQUESTED** [review]
See comments on the currency list

File: core/src/main/java/bisq/core/locale/CurrencyUtil.java (hunk 1 above)
Comment (ghost) [reply core/src/main/java/bisq/core/locale/CurrencyUtil.java]: That's true! Nice catch, I unified the previous list with current list so it stayed there.

It appears Revolut discontinued INR support (https://community.revolut.com/t/revolut-in-india-not-getting-interbank-rates/16999/14).

**Review by sqrrm — APPROVED** [review]
utACK

Please don't force push changes. When reviewing I know I have reviewed the first commit and can easily see what has changed in the second. Now I have to review the entire PR from scratch again.

**Pull request merged**